### DISCUSSION TITLE
Export

### DISCUSSION THREAD

**Discussion by mikekorth** [phcode-dev/phoenix#986] [Ideas]
An "export project" button would be incredibly useful for anyone wanting to use this IDE

**Comment by abose** [comment]
@mikekorth Export is already supported as `Download project`  if the folder is not in a local drive. Or is it something else that you have in mind?

![image](https://user-images.githubusercontent.com/5336369/222911521-de7a9f7e-5296-4e43-a264-84e42f03dec7.png)
![image](https://user-images.githubusercontent.com/5336369/222911666-1cf3654c-c83c-43e5-8c4c-c283e9784e56.png)
